src/main/java/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction.java

```diff
@@ -45,15 +45,15 @@
 			Messages._LockableResourcesRootAction_ReservePermission_Description(), Jenkins.ADMINISTER,
 			PermissionScope.JENKINS);
 
+	public static final Permission VIEW = new Permission(PERMISSIONS_GROUP,
+			Messages.LockableResourcesRootAction_ViewPermission(),
+			Messages._LockableResourcesRootAction_ViewPermission_Description(), Jenkins.ADMINISTER,
+			PermissionScope.JENKINS);
+	
 	public static final String ICON = "/plugin/lockable-resources/img/device-24x24.png";
 
 	public String getIconFileName() {
-		if (User.current() != null) {
-			// only show if logged in
-			return ICON;
-		} else {
-			return null;
-		}
+		return (Jenkins.getInstance().hasPermission(VIEW)) ? ICON : null;
```

Comment:
I think it is safe to use `(Jenkins.getInstance().hasPermission(Jenkins.ADMINISTER)) ? ICON : null;`
And this would be the only change needed.

Reply:
We only allow the ones who have permission to see the management page & delete locks in my company, but it's okay to at least have a null check.